examples/NSIDC/ISat2_from_CMR_to_S3.ipynb

```diff
@@ -0,0 +1,665 @@
+      "Title: ATLAS/ICESat-2 L2A Global Geolocated Photon Data V004\n",
+      "Abstract: This data set (ATL03) contains height above the WGS 84 ellipsoid (ITRF2014 reference frame), latitude, longitude, and time for all photons downlinked by the Advanced Topographic Laser Altimeter System (ATLAS) instrument on board the Ice, Cloud and land Elevation Satellite-2 (ICESat-2) observatory. The ATL03 product was designed to be a single source for all photon data and ancillary information needed by higher-level ATLAS/ICESat-2 products. As such, it also includes spacecraft and instrument parameters and ancillary data not explicitly required for ATL03.\n",
+      "\n",
+      "concept-id: C1705401930-NSIDC_ECS\n",
+      "Title: ATLAS/ICESat-2 L2A Global Geolocated Photon Data V003\n",
+      "Abstract: This data set (ATL03) contains height above the WGS 84 ellipsoid (ITRF2014 reference frame), latitude, longitude, and time for all photons downlinked by the Advanced Topographic Laser Altimeter System (ATLAS) instrument on board the Ice, Cloud and land Elevation Satellite-2 (ICESat-2) observatory. The ATL03 product was designed to be a single source for all photon data and ancillary information needed by higher-level ATLAS/ICESat-2 products. As such, it also includes spacecraft and instrument parameters and ancillary data not explicitly required for ATL03.\n",
+      "\n",
+      "concept-id: C2003771331-NSIDC_ECS\n",
+      "Title: ATLAS/ICESat-2 L3A Land Ice Height V004\n",
+      "Abstract: This data set (ATL06) provides geolocated, land-ice surface heights (above the WGS 84 ellipsoid, ITRF2014 reference frame), plus ancillary parameters that can be used to interpret and assess the quality of the height estimates. The data were acquired by the Advanced Topographic Laser Altimeter System (ATLAS) instrument on board the Ice, Cloud and land Elevation Satellite-2 (ICESat-2) observatory.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from cmr.search import collection as cmr_collection\n",
+    "from cmr.search import granule \n",
+    "from cmr.auth import token\n",
+    "\n",
+    "# NON_AWS collections are hosted at the NSIDC DAAC data center\n",
+    "# AWS_CLOUD collections are hosted at AWS S3 us-west-2\n",
+    "NSIDC_PROVIDERS = {\n",
+    "    'NSIDC_HOSTED': 'NSIDC_ECS', \n",
+    "    'AWS_HOSTED':'NSIDC_CPRD'\n",
+    "}\n",
+    "\n",
+    "# First let's search for some collections hosted at NSIDC using a keyword\n",
+    "collections = cmr_collection.search({'keyword':'ice',\n",
+    "                                     'provider': NSIDC_PROVIDERS['NSIDC_HOSTED']})\n",
+    "\n",
+    "# Let's print some information about the first 3 collection that match our provider\n",
+    "for collection in collections[0:3]:\n",
+    "    print(f\"concept-id: {collection['meta']['concept-id']}\\n\" +\n",
+    "          f\"Title: {collection['umm']['EntryTitle']}\\n\" +\n",
+    "          f\"Abstract: {collection['umm']['Abstract']}\\n\")"
```

Comment:
Suggestion: use the textwrap.wrap function to wrap the abstracts. The textwrap library is part of the standard library that ships with python, so you won't have to install anything new. I haven't run this, so there may be a typo, but you'd do something like:

```
import textwrap

for collection in collections[0:3]:
    wrapped_abstract = "\n".join(textwrap.wrap(f"Abstract: {collection['umm']['Abstract']},60))
    print(f"concept-id: {collection['meta']['concept-id']}\n" +
          f"Title: {collection['umm']['EntryTitle']}\n" +
          f"{wrapped_abstract}\n")

```